src/main/java/com/mraof/minestuck/data/recipe/MinestuckRecipeProvider.java

```diff
@@ -269,6 +269,50 @@ protected void buildRecipes(Consumer<FinishedRecipe> recipeBuilder)
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.POLISHED_CRUXITE_SLAB.get(), 6).define('#', MSBlocks.POLISHED_CRUXITE_BLOCK.get()).pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.POLISHED_CRUXITE_WALL.get(), 6).define('#', MSBlocks.POLISHED_CRUXITE_BLOCK.get()).pattern("###").pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_BRICKS.get(), 4).define('#', MSBlocks.CRUXITE_BLOCK.get()).pattern("## ").pattern("## ").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_BRICK_STAIRS.get(), 4).define('#', MSBlocks.CRUXITE_BRICKS.get()).pattern("#  ").pattern("## ").pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_BRICK_SLAB.get(), 6).define('#', MSBlocks.CRUXITE_BRICKS.get()).pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_BRICK_WALL.get(), 6).define('#', MSBlocks.CRUXITE_BRICKS.get()).pattern("###").pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
+		SimpleCookingRecipeBuilder.smelting(Ingredient.of(MSBlocks.CRUXITE_BLOCK.get()), RecipeCategory.BUILDING_BLOCKS, MSBlocks.SMOOTH_CRUXITE_BLOCK.get(), 0.1f, 200).group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSItems.RAW_CRUXITE.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CHISELED_CRUXITE_BLOCK.get(), 4).define('#', MSBlocks.CRUXITE_BRICKS.get()).pattern("## ").pattern("## ").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_PILLAR.get(), 2).define('#', MSBlocks.SMOOTH_CRUXITE_BLOCK.get()).pattern("#  ").pattern("#  ").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
+		ShapedRecipeBuilder.shaped(RecipeCategory.BUILDING_BLOCKS, MSBlocks.CRUXITE_LAMP.get(), 1).define('#', MSItems.RAW_CRUXITE.get()).define('%', Items.TORCH).pattern("###").pattern("#%#").pattern("###").group("cruxite_blocks").unlockedBy("has_raw_cruxite", has(MSBlocks.WHITE_CHESS_BRICKS.get())).save(recipeBuilder);
```

Comment:
These are using `WHITE_CHESS_BRICKS` for the recipe unlock condition.

Reply:
Not sure how that slipped through, when I used a find/replace... copy paste error? idk I'll update these.